### ISSUE TITLE
Static Asset Caching

### ISSUE THREAD

**Issue by carlosloureda** [alexadark/atsg#25]

**Comment by Renesauve** [comment]
Could use more information on this before I jump in, after running build and running lighthouse report I'm not seeing anything related to "static content caching from JS, CSS, and different image types". Will come back to this one.

**Comment by carlosloureda** [comment]
Yes sure, let me explain better, just copy-pasted Roberto´s words.

- Go to **web.dev** and run an audit against our **master** https://atsgcloud.gatsbyjs.io/.
- There you go to **view report**, my last report is available here: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fatsgcloud.gatsbyjs.io%2F
- There on the **diagnostics** section you see the section: **Serve static assets with an efficient cache policy **

- Let me give you an screenshot:
![image](https://user-images.githubusercontent.com/1656532/119736728-76030c00-be7e-11eb-8d1d-c9e17b41ed60.png)


**NOTE**: Let me know if this a crazy task as from web.dev reports they say that doesn´t interfer in the performance, so if it is very difficult or whatever we can put it on the icebox.
